packages/angular/build/src/utils/i18n-options.ts

```diff
@@ -98,6 +110,15 @@ export function createI18nOptions(
       ensureString(metadata.sourceLocale.baseHref, 'i18n sourceLocale baseHref');
       rawSourceLocaleBaseHref = metadata.sourceLocale.baseHref;
     }
+
+    if (metadata.sourceLocale.urlSegment !== undefined) {
+      ensureValidateUrlSegment(metadata.sourceLocale.urlSegment, 'i18n sourceLocale urlSegment');
```

Comment:
Consider: "i18n sourceLocale urlSegment" is an `angular.json` option right? Would `i18n.sourceLocale.urlSegment` be more clear about the context?

It seems like this format is pre-existing in this file, so maybe we already have a convention here, I'm just not sure I would understand this formatting.

Reply:
Updates